src/components/layout/homepage/CommunityCard.tsx

```diff
@@ -34,7 +34,7 @@ export default function CommunityCard() {
     {
       title: 'Hackathons & Grants',
       description:
-        'Jump-start your project or your career in the blockchain space',
+        'Find out how you can jump-start your project or your career in the blockchain space.',
```

Comment:
I actually prefer the previous, more concise version. It has the same meaning but a stronger voice by starting with a verb.